lib/ansible/plugins/action/nxos.py

```diff
@@ -55,10 +55,23 @@ def run(self, tmp=None, task_vars=None):
                 self._task.args['username'] = self._play_context.connection_user
 
         if self._task.action == 'nxos_install_os':
+            persistent_command_timeout = 0
+            persistent_connect_timeout = 0
             connection = self._connection
-            if connection.get_option('persistent_command_timeout') < 600 or connection.get_option('persistent_connect_timeout') < 600:
+            if connection.transport == 'local':
```

Comment:
@mikewiebe Can you update the module docs which states, if someone set the timeouts as config task variable network_cli/httpapi connection plugin is required. `local` only supports these timeout values with ansible.cfg or env var.

Reply:
Yes, I can do that.